src/browser/ui/dom/CSSPropertyOperations.js

```diff
@@ -64,6 +68,33 @@ if (__DEV__) {
       name.charAt(0).toUpperCase() + name.slice(1) + '?'
     );
   };
+
+  var warnStyleValueWithSemicolon = function(name, value) {
+    if (warnedStyleValues.hasOwnProperty(value) && warnedStyleValues[value]) {
+      return;
+    }
+    
+    warnedStyleValues[value] = true;
+    warning(
+      false,
+      'Style property values shouldn\'t contain a semicolon. Try "' +
+      name + ': ' + value.replace(badStyleValueWithSemicolonPattern, '') + '" instead.'
```

Comment:
How should this be handled if the message itself is > 80 max-len? Ok to concatenate actual message to avoid lint failures?

```js
warning(
  false,
  'Style property values shouldn\'t contain a semicolon. ' +
  'Try "%s: %s" instead.',
  name,
  value.replace(badStyleValueWithSemicolonPattern, '')
);
```